script.allowed_contexts - unknown script context

I have a config file that works file with 6.3.2 but when I try and upgrade to 7.2.0 I get this exception:

```
[2019-07-11T17:26:36,202][WARN ][o.e.b.ElasticsearchUncaughtExceptionHandler] [1node] uncaught exception in thread [main]
org.elasticsearch.bootstrap.StartupException: java.lang.IllegalArgumentException: unknown script context [search] found in setting [script.allowed_contexts].
        at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:163) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Elasticsearch.execute(Elasticsearch.java:150) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.cli.EnvironmentAwareCommand.execute(EnvironmentAwareCommand.java:86) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.cli.Command.mainWithoutErrorHandling(Command.java:124) ~[elasticsearch-cli-7.2.0.jar:7.2.0]
        at org.elasticsearch.cli.Command.main(Command.java:90) ~[elasticsearch-cli-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:115) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Elasticsearch.main(Elasticsearch.java:92) ~[elasticsearch-7.2.0.jar:7.2.0]
Caused by: java.lang.IllegalArgumentException: unknown script context [search] found in setting [script.allowed_contexts].
        at org.elasticsearch.script.ScriptService.<init>(ScriptService.java:209) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.script.ScriptModule.<init>(ScriptModule.java:92) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.node.Node.<init>(Node.java:331) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.node.Node.<init>(Node.java:251) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Bootstrap$5.<init>(Bootstrap.java:221) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Bootstrap.setup(Bootstrap.java:221) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Bootstrap.init(Bootstrap.java:349) ~[elasticsearch-7.2.0.jar:7.2.0]
        at org.elasticsearch.bootstrap.Elasticsearch.init(Elasticsearch.java:159) ~[elasticsearch-7.2.0.jar:7.2.0]
        ... 6 more
```

The pertinent lines are:
```
# Limit scripting. "inline" is necessary for custom scoring.
script.allowed_types: inline
# Only allow scripts to be used for searching
script.allowed_contexts: search
```

If I remove `script.allowed_contexts` line then 7.2.0 starts fine. Except that this is documented as being correct: https://www.elastic.co/guide/en/elasticsearch/reference/7.2/modules-scripting-security.html
No plugins.

--- a/docs/reference/scripting/security.asciidoc
+++ b/docs/reference/scripting/security.asciidoc
@@ -101,7 +101,7 @@ to be `none`.
 
 [source,yaml]
 ----
-script.allowed_contexts: search, update <1>
+script.allowed_contexts: score, update <1>
 ----
-<1> This will allow only search and update scripts to be executed but not
+<1> This will allow only scoring and update scripts to be executed but not
 aggs or plugin scripts (or any other contexts).

